src/ArrayHelper.php

```diff
@@ -241,26 +242,26 @@ public static function getValue($array, $key, $default = null)
 
         if (strpos($key, '.') !== false) {
             foreach (explode('.', $key) as $part) {
-                if (is_array($array)) {
-                    if (!array_key_exists($part, $array)) {
-                        return $default;
-                    }
-                    $array = $array[$part];
-                } elseif (is_object($array)) {
-                    if (!property_exists($array, $part) && empty($array)) {
```

Comment:
I've tried to de-duplicate code and found that this case seems to be not covered by tests. Would you please check if the use-case for it makes sense?